ci/cirrus_wheels.yml

```diff
@@ -1,6 +1,6 @@
 build_and_store_wheels: &BUILD_AND_STORE_WHEELS
   install_cibuildwheel_script:
-    - python -m pip install cibuildwheel==2.12.0
+    - python -m pip install cibuildwheel==2.14.1
```

Comment:
2.15.0 here too?

Reply:
cibuildwheel makes frequent releases, and they're well tested. I think bumping versions to the latest holds little risks. You're seeing 2.14.1 because that's the latest version that was around when I made the original commit.